packages/terraform-modules/src/base/ApplicationECSService.ts

```diff
@@ -485,6 +489,7 @@ export class ApplicationECSService extends Construct {
       tags: this.config.tags,
       dependsOn: ecrRepos,
       provider: this.config.provider,
+      skipDestroy: true,
```

Comment:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition#skip_destroy

Makes it so that every PR doesn't say its destroying a resource if its just a deploy of new code.